rs/ethereum/cketh/minter/BUILD.bazel

```diff
@@ -241,6 +241,8 @@ closure_js_library(
     target_compatible_with = [
         "@platforms//os:linux",
     ],
+    # We don't automatically build this because the build is unreliable.
```

Comment:
nit: here for a lack of a better place: `feat` in the PR title doesn't seem appropriate. Maybe `build`?

Reply:
changed to `chore()`, PR title validation was not happy with `build()`

Reply:
Also ok by me. (build is part of the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), so it should be supported, no?)